subscriber/src/database.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate postgres;
+use postgres::{Connection, TlsMode, Error};
+
+pub struct Database {
```

Comment:
I'd recommend using [Diesel](https://github.com/diesel-rs/diesel) and [R2D2](https://github.com/sfackler/r2d2) to handle database connections and sending queries. This will give you things like database connection retries for free.